doc/Getting-started/index.md

```diff
@@ -103,68 +114,91 @@ To launch the Kaa sanbox on Amazon Elastic Compute Cloud (Amazon EC2), go throug
+9. Click `Launch Instances`.
+The `Launch Status` page will be displayed.
 
-9. Click **Launch Instances**. The **Launch Status** page will be displayed.
+10. Click `View Instances`.
 
-10. Click **View Instances**.
-
-11. After launching Kaa Sandbox instance, go to **<your\_instance\_public\_domain\_name\>:9080/sandbox** or **<your\_instance\_public\_ip\>:9080/sandbox** URL. Public DNS or IP of your instance are available from your instance description.
+11. After launching Kaa Sandbox instance, go to **<your\_instance\_public\_domain\_name\>:9080/sandbox** or **<your\_instance\_public\_ip\>:9080/sandbox** URL.
+Public DNS or IP of your instance are available from your instance description.
 
 <br>
 
 </div></div>
 
+# Launching your first Kaa application
 
-## Kaa Sandbox web UI
-
-Kaa Sandbox web UI provides you with access to Kaa demo projects and some basic Kaa Sandbox configuration capabilities. Once Kaa Sandbox is installed and opened, the web UI is available at the following URL (by default): <http://127.0.0.1:9080/sandbox>.
+Now that you have your Kaa Sandbox instance up and running, let's run a sample application!
 
-### Demo projects
+Kaa *Sandbox web UI* listens on port 9080 (by default) and provides you with access to [sample applications](https://github.com/kaaproject/sample-apps) and some basic Sandbox configuration capabilities.
 
-You can download both source and binary distributions for each demo project. A downloaded binary already contains Kaa SDK that targets current Kaa Sandbox. 
-Thus, if you successfully configure the [networking](#networking) for your SDK, the downloaded application will be able to access it and will work correctly.
+You can download the source code of every sample application.
+Java- and Android-based sample applications can also be downloaded as ready to run .jar or .apk files respectively.
+As you request source or executable files, the Sandbox will generate and embed into your download an [Endpoint SDK library]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/) configured to work with your specific Kaa server instance.
 
 <br>
 
 ![alt Demo projects view](attach/sandbox_demo_projects.png)
 
-## Admin UI
+To download and run your first Kaa sample application, we suggest that you navigate to the `Data collection demos` application bundle and select the sample app programming language of your preference.
+If you want to avoid setting up the toolchain and rather want to see Kaa platform in action with minimum further setup, it is recommended that you choose Java language, for which the Sandbox will be able to provide you with a ready to launch executable .jar file.
+Follow the instructions in the Sandbox to download, run, and test the sample application.
+
+<br>
+
+![alt First application page](attach/sandbox_first_app_view.png)
```

Comment:
Remove `alt` from [alt text](https://en.wikipedia.org/wiki/Alt_attribute). Seems that someone misunderstood markdown documentation which led to all alt text start with word `alt`. That should be fixed.